cmd/otelcontribcol/builder-config.yaml

```
@@ -66,6 +103,7 @@ exporters:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.80.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/mezmoexporter v0.80.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.80.0
+    - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.80.0
```

```suggestion
  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.80.0
```

Looks like this line is indented further than the other lines around it

---

Thanks!